persistence-modules/spring-data-jpa-query-4/src/test/java/com/baeldung/spring/data/jpa/maxvalue/FindingMaxValueIntegrationTest.java

```diff
@@ -0,0 +1,100 @@
+package com.baeldung.spring.data.jpa.maxvalue;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.util.List;
+import java.util.Optional;
+
+@DataJpaTest
+@ContextConfiguration(classes = {FindingMaxValueIntegrationTest.class, EmployeeMaxValueService.class})
+@EntityScan(basePackageClasses = Employee.class)
+@EnableJpaRepositories(basePackageClasses = EmployeeRepository.class)
+public class FindingMaxValueIntegrationTest {
+
+    @Autowired
+    private EmployeeMaxValueService employeeMaxValueService;
+
+    @Autowired
+    private EmployeeRepository employeeRepository;
+
+    private final static Long EXPECTED_VALUE = 800L;
+
+    @BeforeEach
+    void givenEmployees() {
+        // given
+        Employee employee1 = new Employee(1, "John", 400L);
+        Employee employee2 = new Employee(2, "Paul", 800L);
+        Employee employee3 = new Employee(3, "Jack", 600L);
+
+        employeeRepository.saveAll(List.of(employee1, employee2, employee3));
+    }
+
+    @Test
+    void givenEmployees_whenFindMaxUsingDerivedQuery_thenMaxReturned() {
+        Optional<Employee> result = employeeRepository.findTopByOrderBySalaryDesc();
+
+        Assertions.assertThat(result)
```

Comment:
Add the static imports for Assertions. (same elsewhere)
```suggestion
        assertThat(result)
```